packages/account/src/Components/file-uploader-container/__tests__/file-uploader.spec.tsx

```diff
@@ -48,11 +52,11 @@ describe('<FileUploader />', () => {
 
         const file = new File(['hello'], 'hello.png', { type: 'image/png' });
 
-        const input = screen.getByTestId('dt_file_upload_input');
+        const input = screen.getByTestId('dt_file_upload_input') as HTMLInputElement;
```

Comment:
Suggestion:(Instead of of using 'as' can you give type to the variable directly)